client/components/payment-activity/style.scss

```diff
@@ -23,6 +23,19 @@
 				padding: 16px 0 19px;
 			}
 		}
+
+		&__header {
+			.wcpay-filter.components-custom-select-control {
+				.wcpay-filter.components-custom-select-control__menu {
+					max-height: fit-content;
+				}
```

Comment:
Is this custom to our needs, or would this benefit other clients of `InlineLabelSelect`.

Side note (FYI @Jinksi ) – looks like the CSS classes for `InlineLabelSelect` are out of date? (`custom-select-control`)?

Reply:
> the CSS classes for `InlineLabelSelect` are out of date?

`InlineLabelSelect` only extends the existing Gutenberg `custom-select-control` styles by adding the `.wcpay-filter` class.

https://github.com/Automattic/woocommerce-payments/blob/3270d50a369b96f6f75b0eacec532d2240d0d50f/client/components/inline-label-select/style.scss#L1